Makefile

```
@@ -0,0 +1,10 @@
+build_proto:
+	rm -rf githubsearch_proto && mkdir -p githubsearch_proto
+	protoc --proto_path=proto/ --go_out=./githubsearch_proto \
+	--go-grpc_out=./githubsearch_proto \
+	--go_opt=paths=source_relative \
+	--go-grpc_opt=paths=source_relative \
+	proto/github_search.proto
+
+run_tests:
+	go test github.com/panikdkernel/github-search-api-grpc/internal/wrapper 
```

Missing trailing newline. Also this will not work for anyone but your user unless you switch to the "all packages" expansion as indicated.
```suggestion
	go test ./...
```